lib/macho/load_commands.rb

```diff
@@ -225,6 +282,30 @@ def to_i
         @string_offset
       end
     end
+
+    # Represents the contextual information needed by a load command to
+    # serialize itself correctly into a binary string.
+    class SerializationContext
+      # @return [Symbol] the endianness of the serialized string
+      attr_reader :endianness
+
+      # @return [Fixnum] the constant rounding value used to pad the serialized string
```

Comment:
I wonder whether the use of “string” here is going to cause confusion. I'd probably replace that with “load command”.